common/src/main/scala/com/gu/aws/AWSAsyncHandler.scala

```diff
@@ -14,19 +14,25 @@ class AwsAsyncHandler[Request <: AmazonWebServiceRequest, Response](f: (Request,
 
   private val promise = Promise[Response]()
 
-  override def onError(exception: Exception): Unit = exception match {
-    case e: AmazonServiceException =>
-      if (e.getErrorCode == "ThrottlingException") {
-        logger.warn(s"A rate limiting exception was thrown, we may need to adjust the rate limiting in ClientWrapper.scala")
-        Thread.sleep(1000) //Wait for a second and retry
-        f(request, this)
-      } else {
-        promise.failure(exception)
-      }
-    case _ => promise.failure(exception)
+  override def onError(exception: Exception): Unit = {
+    logger.warn(s"Failure from AWSAsyncHandler", exception)
```

Comment:
No need for `s""` if you are doing string interpolation

Reply:
Thanks, good spot